scripts/test/Muon/phase_table_widget/phase_table_presenter_test.py

```diff
@@ -142,16 +142,6 @@ def test_that_new_phase_table_calculated_if_construct_selected(self):
         self.assertEqual(parameters, {'InputWorkspace': 'MUSR22222_raw_data_period_1',
                                        'PhaseTable': 'created_phase_table'})
 
-    @mock.patch('Muon.GUI.Common.phase_table_widget.phase_table_presenter.MuonWorkspaceWrapper')
-    def test_add_phase_quad_to_ADS_does_so_in_correct_location_with_correct_name(self, mock_workspace_wrapper):
```

Comment:
do we have a test that does a similar check?

Reply:
The workspaces are now added to the ADS by the relevant algorithms so this test is no longer needed.